zipline/data/us_equity_pricing.py

```diff
@@ -481,6 +481,13 @@ def _spot_col(self, colname):
             col = self._spot_cols[colname] = self._table[colname]
         return col
 
+    def get_last_traded_dt(self, asset, day):
+        volumes = self._spot_col('volume')
+        ix = self.sid_day_index(asset, day)
+        while True:
+            if volumes[ix] != 0:
```

Comment:
@jbredeche https://github.com/quantopian/zipline/commit/42b95c60eeec8282b78804212d5abea18948942e should fix this, have time to take a look?